tests/migrations/test_operations.py

```diff
@@ -482,6 +482,22 @@ def test_rename_model_with_self_referential_fk(self):
             self.assertFKExists("test_rmwsrf_rider", ["friend_id"], ("test_rmwsrf_rider", "id"))
             self.assertFKNotExists("test_rmwsrf_rider", ["friend_id"], ("test_rmwsrf_horserider", "id"))
 
+    def test_rename_model_with_superclass_fk(self):
+        """
+        Tests the RenameModel operation on model which has a superclass that has a foreign key
```

Comment:
Please wrap docstrings at 79 characters and include a period at the end of the sentence.